typescript/postgis-express/README.md

```diff
@@ -30,55 +30,35 @@ git clone git@github.com:prisma/prisma-examples.git --depth=1
 Install npm dependencies:
 
 ```
-cd prisma-examples/typescript/postgis-express
+
 npm install
```

Comment:
The `cd prisma-examples/typescript/postgis-express` is missing for the "Alternative: Clone repo" path